catroid/src/org/catrobat/catroid/formulaeditor/Operators.java

```diff
@@ -25,9 +25,9 @@
 import android.util.Log;
 
 public enum Operators {
-	LOGICAL_AND(2, true), LOGICAL_OR(1, true), EQUAL(3, true), NOT_EQUAL(4, true), SMALLER_OR_EQUAL(4, true), GREATER_OR_EQUAL(
-			4, true), SMALLER_THAN(4, true), GREATER_THAN(4, true), PLUS(5), MINUS(5), MULT(6), DIVIDE(6), MOD(6), POW(
-			7), LOGICAL_NOT(4, true);
+	LOGICAL_AND(2, true), LOGICAL_OR(1, true), EQUAL(3, true), NOT_EQUAL(4, true), SMALLER_OR_EQUAL(4, true),
+	GREATER_OR_EQUAL(4, true), SMALLER_THAN(4, true), GREATER_THAN(4, true), PLUS(5), MINUS(5), MULT(6), DIVIDE(6),
+	MOD(6), POW(7), LOGICAL_NOT(4, true);
 
 	private static final String TAG = Operators.class.getSimpleName();
 
```

Comment:
Unfortunately  I can't add a comment on the next line...
```private final Integer priority;``` Would a primitive int be sufficient too?